infoschema/tables.go

```diff
@@ -637,6 +638,10 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) [][]types.Da
 	createTimeTp := tablesCols[15].tp
 	for _, schema := range schemas {
 		for _, table := range schema.Tables {
+			collation := table.Collate
+			if collation == "" {
```

Comment:
Which schema table will be in this situation？

Reply:
Could be more specific? I don't think I fully understand your question.

Reply:
What I mean is which table's charset is not empty but collate is empty. Could you give an example?